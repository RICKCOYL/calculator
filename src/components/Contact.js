const Contact = () => (
  <div>
    <h2>Contact</h2>
    <p>Quis officiis id voluptatem</p>
  </div>
);

export default Contact;
