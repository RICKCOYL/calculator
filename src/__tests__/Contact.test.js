import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Contact from '../components/Contact';

describe('Home component test', () => {
  test('should renders welcome text', () => {
    render(<Contact />);
    const text = screen.getByText('Contact');
    expect(text).toBeInTheDocument();
  });

  test('should render the expected output', () => {
    const component = renderer.create(
      <Contact />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
