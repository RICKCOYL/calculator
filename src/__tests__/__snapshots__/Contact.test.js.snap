// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component test should render the expected output 1`] = `
<div>
  <h2>
    Contact
  </h2>
  <p>
    Quis officiis id voluptatem
  </p>
</div>
`;
